localization/yabloc/yabloc_image_processing/src/line_segments_overlay/line_segments_overlay_core.cpp

```diff
@@ -18,7 +18,11 @@
 #include <yabloc_common/cv_decompress.hpp>
 #include <yabloc_common/pub_sub.hpp>
 
+#if __has_include(<cv_bridge/cv_bridge.hpp>)
+#include <cv_bridge/cv_bridge.hpp>
+#else
 #include <cv_bridge/cv_bridge.h>
+#endif
```

Comment:
We might want to leave comments so that we know why we need this when we look back to this code in the future.

```suggestion
#if __has_include(<cv_bridge/cv_bridge.hpp>)
#include <cv_bridge/cv_bridge.hpp>  // for ROS 2 Jazzy
#else
#include <cv_bridge/cv_bridge.h>  // for ROS 2 Humble
#endif
```

Reply:
We should also fix other `__has_include` codes.

Reply:
I see. This comment will help devs later. Thanks!

Similar usage of `__has_include` can be found some files that is not included in this PR.
(These codes seem to have been added to support ROS 2 Iron.)
Should I add some comment to these files?

Reply:
Yes, I think that would help.